README.md

```diff
@@ -133,3 +189,46 @@ rocm_check_target_ids
     )
 
 Returns the subset of HIP [target-ids](https://clang.llvm.org/docs/ClangOffloadBundler.html#target-id) supported by the current CXX compiler.
+
+ROCMClients
+===========
+
+rocm_package_setup_client_component
```

Comment:
I think it would be better to prefix the client functions with `rocm_client` which would make this function `rocm_client_package_setup_component`, but maybe thats not a better name.

Reply:
I think that naming convention would also be fine, my thought process was primarily to name all of the new packaging related functions with `rocm_package`. Also, this function sets up a client component, whereas `rocm_package_setup_component` sets up a generic component, but the packaging is all the same, so having `client` closer to `component` makes slightly more sense to me.

All that said, I'm not particularly attached to this name, so if you think `rocm_client_package_setup_component` would be better I'm happy to change that.

Reply:
Yea, I am not sure if thats a better name, so if you dont think so either then lets leave it as it is.